lib/ecc/src/KeyUtils.js

```diff
@@ -142,7 +142,10 @@ const key = {
         if (sequence < 0) {
             throw new Error("invalid sequence");
         }
-        brainKey = key.normalize_brainKey(brainKey);
+        if (brainKey.trim()==="") {
```

Comment:
Do we need to check whether `brainKey` is `null`, `undefined` or similar?

Reply:
SHOULD be handled here if I'm not mistaken:
https://github.com/bitshares/bitsharesjs/blob/14f8ffc155ea0bb64653d1bbb2236a6f1de85ed2/lib/ecc/src/KeyUtils.js#L154